src/Stacktrace.php

```diff
@@ -18,6 +18,16 @@ class Stacktrace implements \JsonSerializable
 {
     private const INTERNAL_FRAME_FILENAME = '[internal]';
 
+    /**
+     * @var bool Flag indicating whether it's responsibility of this class to
+     *           read the source code excerpts for each frame
+     *
+     * @internal
+     *
+     * @deprecated since version 2.4, to be removed in 3.0
```

Comment:
>Why deprecated?

Because its only purpose is to trigger the deprecation error when users try to fill the context lines of a frame using this class instead of the newer integration.

>why `protected`?

Because the rest of the class methods are `protected` and if someone overridden any of them he may want to access the property, even if it's marked as `internal`. Do you think it's not worth keeping it as is?

Reply:
I'm undecided. Feel free to do as you wish here.